packages/react-scripts/package.json

```
@@ -17,7 +17,8 @@
   },
   "scripts": {
     "prettier": "npx prettier --arrow-parens always --single-quote --trailing-comma es5 --print-width 120 --no-semi --write \"template*/**/*.js\"",
-    "fs-publish": "npmPublish"
+    "fs-publish": "npmPublish",
+    "prepublish": "modernizr -c modernizr-config.json"
```

```suggestion
    "prepare": "modernizr -c modernizr-config.json"
```
`prepublish` is deprecated, replaced by `prepare`. I suggested `prepublishOnly` which is another alternative, but that would not work if we needed this file to exist locally after install but before publish